### PR TITLE
Refine webui

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
- Refine directory

   image

- Use absolute path [`tsconfig.json`, `.eslintrc`, `webpack.config.dev.js`, `webpack.config.js` ]
- Refine name, such as rename logDrawer to logPanel
- Add notes
- Delete repeated code, such as `initializeIcons();`
- Merge fluent style

### How to test ###
run an experiment and see the web portal is normal

### CODE REVIEW

File: ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/searchFunction.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { mergeStyleSets } from '@fluentui/react';
-import { trialJobStatus } from '../../../static/const';
-import { EXPERIMENT } from '../../../static/datamodel';
-import { TableObj, SearchItems } from '../../../static/interface';
+import { trialJobStatus } from '../../../../../../static/const';
```
Comment (ultmaster) [inline ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/searchFunction.ts:2]: Could you try absolute-path import? The import path looks really long and strange.

File: ts/webui/src/components/experiment/overview/Overview.tsx [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
-import { itemStyleSucceed, entriesOption } from './overview/overviewConst';
-import '../static/style/overview/overview.scss';
-import '../static/style/overview/topTrial.scss';
-import '../static/style/logPath.scss';
+import { EXPERIMENT, TRIALS } from '../../../static/datamodel';
+import { Trial } from '../../../static/model/trial';
+import { AppContext } from '../../../App';
+import { Title } from './Title';
+import SuccessTable from './table/SuccessTable';
+import DefaultPoint from '../trialdetail/graph/DefaultMetricPoint';
```
Comment (ultmaster) [inline ts/webui/src/components/experiment/overview/Overview.tsx:8]: Suggest `trialdetail/chart`

File: ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/searchFunction.ts (hunk 1 above)
Comment (liuzhe-lz) [reply ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/searchFunction.ts:2]: https://github.com/microsoft/nni/blob/master/ts/webui/scripts/config/webpack.config.dev.js#L30
https://github.com/microsoft/nni/blob/master/ts/webui/scripts/config/webpack.config.js#L255

Change above two lines to `modules: [ 'node_modules', 'src' ],`

Comment (Lijiaoa) [reply ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/searchFunction.ts:2]: I try to resolve this issue according [the reference](https://juejin.cn/post/6892636779850137608) but not succeed. It's so easy to resolve by change two lines! Thanks for your help. :)

File: ts/webui/src/components/experiment/overview/Overview.tsx (hunk 2 above)
Comment (Lijiaoa) [reply ts/webui/src/components/experiment/overview/Overview.tsx:8]: ok, will use chart!

**Review by ultmaster — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**